### DISCUSSION TITLE
Access to admin/settings doesn't work

### DISCUSSION THREAD

**Discussion by sdkane** [Athou/commafeed#1533] [Q&A]
I am trying to host a new insurance of commafeed using the latest-h2 document container. I cannot access the public url setting because I cannot access the admin settings. As I understand it this should be accessible under 127.0.0.1:8082/#/admin/settings or /#/app/admin/settings however both redirect to /#/app/categories/all. I have tried signed in on the default admin account and a newly created admin account. Has this moved with the 5.0 update? Or am I doing something wrong?

**Comment by Athou** [comment]
The `publicUrl` setting has been removed in 5.0.0, it can now be deduced from the context automatically when needed.

Also, what version are you upgrading from? I think the admin settings page was made read-only in 2.0.0, 10 years ago and completely removed in 3.0.0, 1.5 years ago. Please read the [README](https://github.com/Athou/commafeed?tab=readme-ov-file#configuration) for instructions about CommaFeed configuration.

**Comment by sdkane** [comment]
I appreciate your answer. I did read the readme and I looked through all the optional settings before asking this question.

I am not upgrading, as I mentioned I am hosting a new instance 

Do you have an example nginx config for proxying commafeed? Because if the issue isn't in the public url setting it must be there for me. I am getting a "resource not found" page when using the following nginx config.

```
location /commafeed {
    proxy_pass http://127.0.0.1:8082;
    proxy_set_header Host $http_host;
    proxy_buffering off;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_set_header X-Forwarded-Proto $scheme;     
    proxy_set_header X-Forwarded-Host $server_name;
}
```

**Comment by sdkane** [comment]
I resolved the issue with nginx proxy_pass. The following configuration is what worked for me.
Note the additional forward slash after the location and the proxy_pass as compared to my earlier attempt.
```
location /commafeed/ {
        proxy_pass http://127.0.0.1:8082/;
        proxy_set_header Host $host;
        proxy_buffering off;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header X-Forwarded-Proto $scheme;
}
```
